packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -69,28 +62,65 @@ const SuggestedActions = ({ className, suggestedActions = [] }) => {
     />
   ));
 
-  if (suggestedActionLayout === 'stacked') {
-    return (
-      <div aria-label=" " aria-live="polite" role="status">
+  const renderSuggestedActionsEmpty = useCallback(() => <ScreenReaderText text={suggestedActionsContainerText} />, [
```

Comment:
I prefer `renderEmptySuggestedActions` better. The other two should follow the same pattern. Honestly, doesn't really matter though.